css/bootstrap.css

```diff
@@ -2108,7 +2106,7 @@ pre code {
     width: 75%;
   }
   .col-lg-8 {
-    width: 66.66666667%;
```

Comment:
Don't change bootstrap! This will mess up other's codes!
Override css at top in index.html !